lib/tools/system-calls.js

```diff
@@ -386,7 +386,10 @@ let isExecLocked = false;
  *                        You can also set the additional `exclusive` param
  *                        to `true` that assures no other parallel adb commands
  *                        are going to be executed while the current one is running
- * @return {string} - Command's stdout.
+ *                        You can set the `outputFormat` param to `stdout` to receive just the stdout
+ *                        output (default) or `full` to receive the entire stdout, stderr, and code
+ *                        response from a command with a zero exit code
+ * @return {string|Object} - Command's stdout or an object containing stdout, stderr, and code.
```

Comment:
I would rather prefer to have a typedef for the alternative result: https://github.com/appium/appium-adb/pull/574/files#diff-3ea7ac5fe8ae9a5687f7df112cce745b55e4952185827ecd1314a69561f38d5dL201

Reply:
Good idea! Added `ExecResult` typedef containing `stdout` and `stderr` (no `code` per your other comment)